api/protos/bootstrap/v1/bootstrap.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package bootstrap;
+option go_package = "bootstrap/v1;bootstrapv1";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+
+
+// [#next-free-field: 4]
+message Bootstrap {
+    // xds-relay server settings.
+    Server server = 1;
+
+    // Configuration information about the control plane/upstream server.
+    Cluster control_plane_cluster = 2;
+
+    // Logging settings.
+    Logging logging = 3;
+
+    // Request/response cache settings.
+    Cache cache = 4;
+}
+
+// [#next-free-field: 2]
+message Server {
+    // The TCP address that the xds-relay server will listen on.
+    SocketAddress address = 1;
+}
+
+// [#next-free-field: 2]
+message Cluster {
+    // The address for the cluster.
+    SocketAddress address = 1;
+}
+
+// [#next-free-field: 3]
+message Logging {
+    // Filepath where logs are emitted. If no filepath is specified, logs will be written to `/dev/null`.
+    string path = 1;
+
+    // The logging level. If no logging level is set, the default is `info`.
+    string level = 2;
```

Comment:
Could we enforce this to be one of `INFO`, `DEBUG`, `WARN`, or `ERROR`?

Reply:
I limited the values by creating an enum in the proto definition - should we limit the log levels exposed to the user via CLI as well? zap has the following:
```
	// DebugLevel logs are typically voluminous, and are usually disabled in
	// production.
	DebugLevel = zapcore.DebugLevel
	// InfoLevel is the default logging priority.
	InfoLevel = zapcore.InfoLevel
	// WarnLevel logs are more important than Info, but don't need individual
	// human review.
	WarnLevel = zapcore.WarnLevel
	// ErrorLevel logs are high-priority. If an application is running smoothly,
	// it shouldn't generate any error-level logs.
	ErrorLevel = zapcore.ErrorLevel
	// DPanicLevel logs are particularly important errors. In development the
	// logger panics after writing the message.
	DPanicLevel = zapcore.DPanicLevel
	// PanicLevel logs a message, then panics.
	PanicLevel = zapcore.PanicLevel
	// FatalLevel logs a message, then calls os.Exit(1).
	FatalLevel = zapcore.FatalLevel
```

Reply:
I think that'd be a good idea, if the error isn't propagated already during parsing of the proto. Our logger supports all the above except `DpanicLevel`

Reply:
IMO we should catch this error during parsing of the proto.

Reply:
The proto will limit the options in the config to INFO/DEBUG/WARN/ERROR, but I meant whether we should prevent the user from entering other log levels via CLI.